keps/prod-readiness/sig-scheduling/3990.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3990
+beta:
+  approver: "@wojtek-t"
```

Comment:
@wojtek-t I assigned it to you as other sig-scheduling KEPs do. 
But, please reassign it to other people if needed. 🙏